RadarSDK/RadarSdkConfiguration.m

```diff
@@ -0,0 +1,43 @@
+//
+//  RadarSDKConfiguration.m
+//  RadarSDK
+//
+//  Copyright © 2023 Radar Labs, Inc. All rights reserved.
+//
+
+#import "RadarSDKConfiguration.h"
+
+#import "RadarLog.h"
+
+@implementation RadarSDKConfiguration
+
+- (instancetype)initWithLogLevel:(RadarLogLevel)logLevel {
+    if (self = [super init]) {
+        _logLevel = logLevel;
+    }
+    return self;
+}
+
++ (RadarSDKConfiguration *_Nullable)sdkConfigurationFromDictionary:(NSDictionary *)dict {
+    if (!dict) {
+        return NULL;
```

Comment:
```suggestion
        return nil;
```

Reply:
(It'll be helpful to check the codebase for how to implement certain patterns when you're writing SDK code — I know objective C is a bit unfamiliar, lean on the prior art)

Reply:
this is because I'm professional C programmer, and NULL didn't raising any errors so... (it shall be fixed)